integration-tests/load/automationv2_1/automationv2_1_test.go

```diff
@@ -124,6 +131,24 @@ Password = '%s'`
 	}
 )
 
+func setUpDataStreamsWireMock(url string) error {
+	wm := gowiremock.NewClient(url)
+	rule200 := gowiremock.Get(gowiremock.URLPathEqualTo("/api/v1/reports/bulk")).
+		WithQueryParam("feedIDs", gowiremock.EqualTo("0x000200")).
+		WillReturnResponse(gowiremock.NewResponse().
+			WithBody(`{"reports":[{"feedID":"0x000200","validFromTimestamp":0,"observationsTimestamp":0,"fullReport":"0x000abc"}]}`).
+			WithStatus(200))
+	err := wm.StubFor(rule200)
+	if err != nil {
+		return err
+	}
+	resp, err := http.Post(fmt.Sprintf("%s/__admin/mappings/save", url), "application/json", nil)
```

Comment:
You might need to change this slightly to:

```
resp, err := http.Post(fmt.Sprintf("%s/__admin/mappings/save", url), "application/json", nil)
if err != nil {
    return errors.New("error saving wiremock mappings")
}
defer resp.Body.Close()

if resp.StatusCode != 200 {
    return errors.New("error saving wiremock mappings")
}

return nil
```

Even though you don't need to use the response body, I think you still need to close it in the event that you get a nil error back from making the request?